.travis.yml

```diff
@@ -2,6 +2,7 @@ language: go
 
 go:
   - "1.10.x"
+  - "1.11.x"
```

Comment:
Since its released,  can you add 1.12 too?

Reply:
Sure, will do.